tutorials/napps/how_to_protect_a_napp_rest_endpoint.rst

```diff
@@ -74,94 +74,205 @@ Now, you must input some data about your NApp:
     --------------------------------------------------------------
 
     Please, insert your NApps Server username: <username>
-    Please, insert your NApp name: hellorest
-    Please, insert a brief description for your NApp [optional]: Hello, world!
+    Please, insert your NApp name: authTest
+    Please, insert a brief description for your NApp [optional]: authTest napp!
 
     Congratulations! Your NApp have been bootstrapped!
-    Now you can go to the directory <username>/hellorest and begin to code your NApp.
+    Now you can go to the directory <username>/authTest and begin to code your NApp.
     Have fun!
 
 After filling your data, you can access the base code of your application in main.py:
 
+You can open the main.py file with any code editor of your preference. 
+
 .. code-block:: console
 
     $ cd ~/tutorials
-    $ gedit <username>/hellorest/main.py
+    $ <code-editor> <username>/authTest/main.py
+
+You can use nano, vi, emacs, gedit, vscode, atom or another code editor, replace 
+<code-editor> with the code editor. See the example using nano:
+
+.. code-block:: console
+
+    $ nano <username>/authTest/main.py
+
 
 The base code will initially look like this:
 
 .. code-block:: python
 
+    """Main module of <username>/authTest Kytos Network Application.
+
+    authTest napp !
+    """
+
     from kytos.core import KytosNApp, log
-    from napps.<username>.hellorest import settings
+
+    from napps.<username>.authTest import settings
 
 
     class Main(KytosNApp):
+        """Main class of <username>/authTest NApp.
+
+        This class is the entry point for this napp.
+        """
 
         def setup(self):
+            """Replace the '__init__' method for the KytosNApp subclass.
+
+            The setup method is automatically called by the controller when your
+            application is loaded.
+
+            So, if you have any setup routine, insert it here.
+            """
             pass
 
         def execute(self):
+            """Run after the setup method execution.
+
+            You can also use this method in loop mode if you add to the above setup
+            method a line like the following example:
+
+                self.execute_as_loop(30)  # 30-second interval.
+            """
             pass
 
         def shutdown(self):
-            pass
+            """Run when your napp is unloaded.
```

Comment:
Please the standard "NApp" (with both first letters in uppercase in all the comments and docstrings). Change it in the NApp template if needed.